src/serialize.js

```diff
@@ -27,7 +27,7 @@ const removeDuplicateIncludes = included => {
 
 const extractResourceInformation = (resource, attributeNames, relationshipNames) => {
   const identifier = extractIdentifier(resource)
-  const permittedAttributes = pick(resource, attributeNames)
+  const permittedAttributes = isEmpty(attributeNames) ? resource : pick(resource, attributeNames)
```

Comment:
Wdyt about this, so you can explicitly use an empty set of attributes if you want?

```suggestion
  const permittedAttributes = attributeNames == null ? resource : pick(resource, attributeNames)
```

Reply:
That's a nice idea, yea.